3723/0/about_dice_project.rb

```diff
@@ -0,0 +1,68 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class DiceSet
+  attr_accessor :values
+
+  def roll(num)
+    @values = []
+    num.times do
+      @values << rand(1..6)
```

Comment:
can you do this during array init?

Reply:
We have 'test_dice_values_should_change_between_rolls' method in this koan. So I have to do the implementation like this.

Reply:
Fixed